css/media.css

```diff
@@ -0,0 +1,110 @@
+@media (max-width:1080px){
+    .social-network{
```

Comment:
```suggestion
    .social-network {
```
не забывай про пробелы после селектора, пройдись везде и поправь

Reply:
Выполнил